### ISSUE TITLE
Rustfmt is removing comments

### ISSUE THREAD

**Issue by GuillaumeGomez** [rust-lang/rustfmt#4541]
A good example:

```
$ cargo fmt
$ git diff
git diff
diff --git a/atk/sys/src/lib.rs b/atk/sys/src/lib.rs
index bce5341b6..9f32aab56 100644
--- a/atk/sys/src/lib.rs
+++ b/atk/sys/src/lib.rs
@@ -1761,10 +1761,6 @@ impl ::std::fmt::Debug for AtkWindow {
 
 #[link(name = "atk-1.0")]
 extern "C" {
-
-    //=========================================================================
-    // AtkCoordType
-    //=========================================================================
     pub fn atk_coord_type_get_type() -> GType;
 
     //=========================================================================
diff --git a/gdk-pixbuf/sys/src/lib.rs b/gdk-pixbuf/sys/src/lib.rs
index 67d190558..9d7b5ebc8 100644
--- a/gdk-pixbuf/sys/src/lib.rs
+++ b/gdk-pixbuf/sys/src/lib.rs
@@ -166,10 +166,6 @@ impl ::std::fmt::Debug for GdkPixbufSimpleAnimIter {
 
 #[link(name = "gdk_pixbuf-2.0")]
 extern "C" {
-
-    //=========================================================================
-    // GdkColorspace
-    //=========================================================================
     pub fn gdk_colorspace_get_type() -> GType;
 
     //=========================================================================
diff --git a/gdk/sys/src/lib.rs b/gdk/sys/src/lib.rs
index 26e5cee34..9fa1dcd15 100644
--- a/gdk/sys/src/lib.rs
+++ b/gdk/sys/src/lib.rs
@@ -4088,10 +4088,6 @@ impl ::std::fmt::Debug for GdkDevicePad {
 
 #[link(name = "gdk-3")]
 extern "C" {
-
-    //=========================================================================
-    // GdkAxisUse
-    //=========================================================================
     pub fn gdk_axis_use_get_type() -> GType;
 
     //=========================================================================
diff --git a/gdkx11/sys/src/lib.rs b/gdkx11/sys/src/lib.rs
index 6518a3a4d..59c5b4b5d 100644
--- a/gdkx11/sys/src/lib.rs
+++ b/gdkx11/sys/src/lib.rs
@@ -255,10 +255,6 @@ impl ::std::fmt::Debug for GdkX11Window {
 
 #[link(name = "gdk-3")]
```

From the repository https://github.com/gtk-rs/gtk-rs

**Comment by calebcartwright** [comment]
Guessing this is a duplicate of #4528, which version of rustfmt are you using?

**Comment by GuillaumeGomez** [comment]
This one:

```
$ rustfmt --version
rustfmt 1.4.26-nightly (293d7d0 2020-11-14)
```

**Comment by calebcartwright** [comment]
Yup, duplicate of #4528 so going to close. It's resolved in 1.4.27 which is available on nightly already (starting on 2020-11-17 IIRC)
